### ISSUE TITLE
Low priority manipulation tasks

### ISSUE THREAD

**Issue by dsherret** [dsherret/ts-simple-ast#27]
I'm going to use this issue to collect together some low priority manipulation tasks. I'll come back to implement these at the end.

**ClassDeclaration**

- ~~Class methods in an ambient context~~ Moved to #125.
- [x] Get & set accessors

**Decorator**

- ~~Renaming~~ - Edit: Won't do. Users can get the appropriate identifier and rename that.
- [x] Setting as a factory or not
- [x] Adding/inserting arguments

**JSDoc**

- [x] Setting description

**Function/Methods/Constructors**

- [x] Setting body text

**VariableDeclarationList**

- [x] Setting declaration type.

**VariableStatement**

- [x] Adding variable declarations

More to come...

**Comment by reda-alaoui** [comment]
@dsherret ,

First, thank you for this amazing lib.

I am building a tool to upgrade an legacy javascript code to typescript. 
Among the conversions, the tool must replace `var` by `let` declaration types everywhere.
I stumbled on:
```
Error: Filling variable declaration type not implemented. Please comment on issue #27 if you need this and I will increase the priority.
```
So here I am asking to increase `Setting declaration type priority`.
I can also make a PR if you give me some guidance.

**Comment by dsherret** [comment]
@reda-alaoui thanks, I'm glad it's useful for you!

It's very easy for me to add that. I'll start work on that now.

**Comment by reda-alaoui** [comment]
Great, thank you :)

**Comment by dsherret** [comment]
@reda-alaoui this is implemented and available in v0.85.0.

I'm going to keep working on the other tasks in this issue.

**Comment by reda-alaoui** [comment]
You rock !
